portality/formcontext/forms.py

```diff
@@ -528,3 +528,32 @@ class ReadOnlyJournalForm(JournalInformation, JournalSubject, JournalLegacy, Not
         * Notes - repeatable notes field
     """
     pass
+
+class ManEdBulkEditJournalForm(Form):
+    publisher = StringField('Publisher',
+        [validators.Optional()]
+    )
+    doaj_seal = DOAJSelectField('Qualifies for Seal',
+        [validators.Optional()],
+        description='How should we change the DOAJ Seal on these journals?',
+        choices=[("", "Leave unchanged"), ("True", "Yes"), ("False", "No")],
+    )
+    country = DOAJSelectField('Country',
+        [validators.Optional()],
+        description='Select the country where the publisher carries out its business activities.',
+        choices=Choices.country(),
+    )
+    owner = StringField('Owner',
+        [ReservedUsernames(), validators.Optional()],
+        description='DOAJ account to which the application belongs.'
```

Comment:
Hmm, I did not see that. @richard-jones can you change it (+ the kwargs thing above), or I can pick it up?